test/suite/test_hs11.py

```diff
@@ -35,15 +35,16 @@
 class test_hs11(wttest.WiredTigerTestCase):
     conn_config = 'cache_size=50MB,statistics=(all)'
     session_config = 'isolation=snapshot'
-    key_format_values = (
+    key_format_values = [
+        ('column', dict(key_format='r')),
         ('int', dict(key_format='i')),
```

Comment:
I know that you didn't add this line of code. @keithbostic was mentioned in our local message channel to not use `int` indicator for row store. As you are modifying this code, how about changing it into `integer_row` or simply `row`?

Reply:
Right, Keith mentioned this to me also, about to fix them all. (int -> integer_row, string -> string_row)

Reply:
Pushed this.